aep/general/0111/aep.md

```diff
@@ -0,0 +1,66 @@
+# Planes
+
+Resources and methods on an API can be divided into the _plane_ that they
+reside or perform operations upon. For the context of APIs, the following
+planes are defined:
+
+- Management plane: a uniform, resource-oriented API that primarily configures
+  and allows retrieval of resources.
+- Data plane: a heterogenous API (ideally resource-oriented) that reads and
+  write user data. Often connects to entities provisioned by the management
+  plane, such as virtual machines.
+
+The term "plane" was originally used in networking architecture. Although
+system and network architecture often defines additional planes (e.g. control
```

Comment:
```suggestion
system and network architecture often defines additional plane (e.g. control
```

If "system and network architecture" is short for "system architecture and network architecture", which I think it is?

Reply:
yep! That's correct. 

I'm going to remove this section though - I don't think it's particularly relevant information, although maybe context is helpful?